Just wanting to double check, following code block is missing from the current version: 

```
> # Export Go path variables
> 			G_EXEC mkdir -p /mnt/dietpi_userdata/go/{bin,pkg,src}
> 			cat << '_EOF_' > /etc/bashrc.d/go.sh
> #!/bin/dash
> export GOPATH=/mnt/dietpi_userdata/go
> export PATH="$PATH:/usr/local/go/bin:$GOPATH/bin"
> _EOF_
> 			# shellcheck disable=SC1091
> 			. /etc/bashrc.d/go.sh
```

--- a/CHANGELOG.txt
+++ b/CHANGELOG.txt
@@ -7,6 +7,7 @@ Enhancements:
 Bug fixes:
 - DietPi-Software | Navidrome: Resolved an issue where on reinstalls the config was was overwritten. Many thanks to @pedrom34 for reporting this issue: https://github.com/MichaIng/DietPi/issues/5840
 - DietPi-Software | Home Assistant: Resolved an issue where some modules didn't work on ARMv6 and ARMv7 systems because of missing runtime libraries. Many thanks to @mail2rst and @HyperCriSiS for reporting this issue: https://dietpi.com/forum/t/after-the-update-homeassistant-mqtt-broker-integration-broken/14719, https://dietpi.com/forum/t/home-assistant-issues-after-latest-upate/14749
+- DietPi-Software | Go: Resolved an issue, introduced with DietPi v8.9, where the Go compiler and development tools were not added to PATH. Many thanks to @envious_jag for reporting this issue: https://dietpi.com/forum/t/installation-snowflake/6566/10
 
 As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: https://github.com/MichaIng/DietPi/pull/XXXX
 

